packages/cascara/src/ui/Table/atoms/SelectionToggle.js

```diff
@@ -2,56 +2,29 @@ import React, { useCallback, useContext } from 'react';
 import pt from 'prop-types';
 
 import CheckBox from './Checkbox';
-import TableContext from '../context/TableContext';
+import { ModuleContext } from '../../../modules/context';
 
 const propTypes = {
   id: pt.string,
 };
 
 const SelectionToggle = ({ id }) => {
```

Comment:
@lordpixel I am pretty sure there is a problem with us using a module for row selection. That will effectively place the checkbox for selecting a row into the form and therefore, we now have a field that could theoretically be submitted in an API response. We will basically break functionality for submitting forms if a table has selections enabled.

Did we validate that these checkboxes are not being included when form submission takes place?

IMO it is going to be _significantly_ safer to just have a basic html checkbox that allows selection and include that at the beginning of the table row.

Maybe I am just reacting to the context being used. Let's confirm that these checkboxes do not show up when submitting forms.

Reply:
@brianespinosa the `checkbox` we are using for selection is NOT the `Checkbox Module` but a simple HTML element. Whilst it is part of the Form as per your suggestion, it is not included when the Form is posted. We can confirm that because the name of the checkbox is the `eID` value, and it is not present in the data posted by the form.